semgrep/tests/e2e/snapshots/test_dependency_aware_rules/test_dependency_aware_rules/rulesdependency_awarelog4shell.yaml-dependency_awarelog4shell.java/results.json

```diff
@@ -0,0 +1,102 @@
+{
+  "errors": [],
+  "paths": {
+    "_comment": "<add --verbose for a list of skipped paths>",
```

Comment:
Those elements are annoying in test files. Why do we have those _comment?

Reply:
I don't know. When I made the PR I got that github-actions request to accept the snapshot changes, and I think it added those.